templates/group.html

```diff
@@ -1 +1 @@
-<a ng-href="#/{{ctrl.group}}">{{ctrl.group}} ({{ctrl.groupCount}})</a>
+<a ng-href="#/{{ctrl.group}}">{{ ctrl.group }} ({{ ctrl.groupCount | counterFormatter }})</a>
```

Comment:
In order to get right-aligning instead of parentheses I gather. Sure thing, fixed.